### ISSUE TITLE
Redesign a user's private home page/home screen

### ISSUE THREAD

**Issue by timbl** [solid/node-solid-server#889]
When a user is logged on they should a great starting home page, different from the public the page which everyone else sees sees when the click .

- Access to the core things everyone starts off with: public folder, address book
- Access to all the things they have already made 
- Access to a set of apps/facilities which the solid project would like to point  out to them 
- A time line view of recent stuff they have been involved in
- Incoming  notifications/requests/opportunities

It must be user configurable but never to the point of block yourself in a place you an't get out of.

Idea: a summary of the public page others see down one side.

**Comment by kjetilk** [comment]
Yeah, this is important. I'm just thinking that this is something we should decouple from the server itself, that it is an app on its own. It could be distributed as a dependency of the server, but not as part of the same repo and the same npm package. That way, it could have independent evolution. What do you think about that?

**Comment by RubenVerborgh** [comment]
So we want the front page to be an app, rather than something static.

I fully agree, but this has quite some codebase consequences for the current NSS.

I agree with @kjetilk that we should decouple the front-end part of the server from the backend, this will also give us compilation pipelines etc., solving some of our other issues. However, I don't think one should be a dependency of the other, because the server might work with multiple front-ends.

Another more radical approach is to ship the server as API-only, and to see everything as an app. But the codebase structure is totally not ready for that.

**Comment by steven-tomlinson** [comment]
FWIW: @rubenverborgh is on the right track in my opinion.
I am thinking about the "Panes" app specifically in this context. I am probably confused but I was under the impression that the Panes app was intended to address this type of requirement.
Is that not the case?

**Comment by kjetilk** [comment]
Right. No strong opinions on the dependency argument here. In Debian package semantics, there's also "recommends", "suggests" and "enhances", pairing the latter might be better, BTW.

Also, I think "everything as an app" (EaaA hereafter) is the direction that @melvincarvalho wants to go too, if I have understood him correctly.

I think that EaaA is feasible for v.next, but not earlier. However, I can also see the urgency, so perhaps what we should look into now is a relatively light-weight homepage for 5.0.0?

**Comment by RubenVerborgh** [comment]
What we could do for v5, is create a new repository for the homepage (with compilation pipeline and everything), and just include a static export in the server repo for now. Or we could link it similarly to how we link solid-auth-client and mashlib.

**Comment by melvincarvalho** [comment]
@kjetilk no strong view.  The idea originally was to develop a dashboard app.  The index page was put together in a few hours as a stop gap.
